Pomodoro: Add commands

jesse-c opened the pull request:
## Description

Adds commands for a [Pomodoro daemon](https://github.com/jesse-c/y-pomodoro) written in Rust that communicates over sockets.

## Type of change

- [x] New script command

## Screenshot

<img width="739" alt="image" src="https://user-images.githubusercontent.com/1405676/130296869-686802e2-3b2d-4108-891b-98619c90d1f6.png">

## Dependencies / Requirements

The [Pomodoro daemon](https://github.com/jesse-c/y-pomodoro) needs to be running.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

unnamedd commented on commands/apps/pomodoro/pause.sh:
```diff
@@ -0,0 +1,17 @@
+
+# Optional parameters:
+# @raycast.icon 🍅
+# @raycast.description A simple Pomodoro timer using https://github.com/jesse-c/y-pomodoro
```
Two things here: 
the first one is that will be nice to have the name of your lib wrapped by a link. The readme will respect if you do in the way I’m suggesting.

second, I couldn’t access your _y-pomodoro_, it doesn’t seem to be public.

```suggestion
# @raycast.description Pause a pomodoro using  [y-pomodoro](https://github.com/jesse-c/y-pomodoro)
```

unnamedd commented on commands/apps/pomodoro/resume.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Optional parameters:
+# @raycast.icon 🍅
+# @raycast.description A simple Pomodoro timer using https://github.com/jesse-c/y-pomodoro
```
```suggestion
# @raycast.description Resumes a pomodoro timer using [y-pomodoro](https://github.com/jesse-c/y-pomodoro)
```

unnamedd commented on commands/apps/pomodoro/pause.sh:
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```
Let’s add the dependency section here and in the other scripts:
https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-runtimes-and-dependencies

jesse-c replied:
Done :)

unnamedd commented on commands/apps/pomodoro/pause.sh:
```diff
@@ -0,0 +1,17 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Pause Pomodoro 
```
```suggestion
# @raycast.title Pause
# @raycast.packageName Pomodoro
```

unnamedd commented on commands/apps/pomodoro/resume.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Resume Pomodoro
```
```suggestion
# @raycast.title Resume
# @raycast.packageName Pomodoro
```

unnamedd commented on commands/apps/pomodoro/show.sh:
```diff
@@ -0,0 +1,17 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Show Pomodoro
```
```suggestion
# @raycast.title Show
# @raycast.packageName Pomodoro
```

unnamedd commented on commands/apps/pomodoro/show.sh:
```diff
@@ -0,0 +1,17 @@
+
+# Optional parameters:
+# @raycast.icon 🍅
+# @raycast.description A simple Pomodoro timer using https://github.com/jesse-c/y-pomodoro
```
```suggestion
# @raycast.description Show active pomodoro timer using [y-pomodoro](https://github.com/jesse-c/y-pomodoro)
```

unnamedd commented on commands/apps/pomodoro/start.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Start Pomodoro
```
```suggestion
# @raycast.title Start
# @raycast.packageName Pomodoro
```

unnamedd commented on commands/apps/pomodoro/start.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Optional parameters:
+# @raycast.icon 🍅
+# @raycast.description A simple Pomodoro timer using https://github.com/jesse-c/y-pomodoro
```
```suggestion
# @raycast.description Start a pomodoro timer using [y-pomodoro](https://github.com/jesse-c/y-pomodoro)
```

unnamedd commented on commands/apps/pomodoro/stop.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Stop Pomodoro
```
```suggestion
# @raycast.title Stop
# @raycast.packageName Pomodoro
```

unnamedd commented on commands/apps/pomodoro/stop.sh:
```diff
@@ -0,0 +1,16 @@
+
+# Optional parameters:
+# @raycast.icon 🍅
+# @raycast.description A simple Pomodoro timer using https://github.com/jesse-c/y-pomodoro
```
```suggestion
# @raycast.description Stop a pomodoro timer using [y-pomodoro](https://github.com/jesse-c/y-pomodoro)
```

unnamedd reviewed (changes requested):
Hey @jesse-c,

Nice addition! I thought we already had a pomodoro timer in the repo, but looks I’m wrong. You are the first one to introduce it to us! 

I made some comments, please address them and let me know when you finish for a second round of review.

To finish, as you are using your own pomodoro implementation, I suggest you to rename the folder from `pomodoro` to `y-pomodoro`, thats will avoid anything kind of possible future conflict.

Thanks for your contribution, pal!

jesse-c commented:
Thanks for the review! I've addressed all the comments. :)

There actually is a set of [Pomodoro scripts](https://github.com/raycast/script-commands/tree/master/commands#pomodoro) from Thomas. 😅

unnamedd commented on commands/apps/y-pomodoro/pause.sh:
```diff
@@ -0,0 +1,20 @@
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Pause
```
This one still missing the `packageName`

```suggestion
# @raycast.title Pause
# @raycast.packageName Pomodoro
```

unnamedd reviewed (changes requested):
@jesse-c just one last thing here, probably I missed the packageName in one of the Script Commands, but the suggestion is now there! :D
